distributedlog-core/src/main/java/com/twitter/distributedlog/util/Utils.java

```diff
@@ -604,4 +604,35 @@ public static void closeQuietly(@Nullable AsyncCloseable closeable) {
                 executorService).map(VoidFunctions.LIST_TO_VOID_FUNC);
     }
 
+    /**
+     * Gets the parent of a path.
+     *
+     * @param path
+     *            path to get the parent of
+     * @return parent of the path or null if no parent exists.
+     */
+    public static String getParent(final String path) {
```

Comment:
could we add zk or unix to the name? 

ex. getZkParent, just to be really clear